blockstore/badger/blockstore_test.go

```diff
@@ -98,6 +99,7 @@ func openBlockstore(optsSupplier func(path string) Options) func(tb testing.TB,
 }
 
 func testMove(t *testing.T, optsF func(string) Options) {
+	ctx := context.TODO()
```

Comment:
This is a test, so Background is fine
```suggestion
	ctx := context.Background()
```